packages/manager/apps/container/src/container/nav-reshuffle/onboarding/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import useOnboarding from '@/core/onboarding';
+
+import OnboardingIntroduction from './introduction';
+import OnboardingWalkMe from './walkMe';
+
+export const NavReshuffleOnboardingWidget = () => {
+  const { isWalkMeVisible } = useOnboarding();
+
+  return (
+    <>
+      {!isWalkMeVisible && <OnboardingIntroduction />}
+      {isWalkMeVisible && <OnboardingWalkMe />}
```

Comment:
```
{ isWalkMeVisible ? <OnBoardingWalkMe /> : <OnboardingIntroduction />
```

Also, since the condition dictates a whole new component to render, it might be worth considering the `useMemo` if the `isWalkMeVisible` is statefull